code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -783,3 +783,60 @@
 	var/location = get_turf(holder.my_atom)
 	for(var/i in 1 to created_volume)
 		new /obj/item/stack/sheet/ashresin(location)
+
+/datum/chemical_reaction/sulfur_trioxide
+	name = /datum/reagent/sulfur_trioxide
+	id = /datum/reagent/sulfur_trioxide
+	results = list(/datum/reagent/sulfur_trioxide = 2)
+	required_reagents = list(/datum/reagent/sulfur = 1, /datum/reagent/oxygen = 3)
+	required_catalysts = list(/datum/reagent/hydrogen = 1)
+
+/datum/chemical_reaction/sulfuric_acid
+	name = /datum/reagent/toxin/acid
+	id = /datum/reagent/toxin/acid
+	results = list(/datum/reagent/toxin/acid = 1)
+	required_reagents = list(/datum/reagent/sulfur_trioxide = 1, /datum/reagent/water = 1)
+
+/datum/chemical_reaction/sugar
+	name = /datum/reagent/consumable/sugar
+	id = /datum/reagent/consumable/sugar
+	results = list(/datum/reagent/consumable/sugar = 1)
+	required_reagents = list(/datum/reagent/carbon = 1, /datum/reagent/oxygen = 1, /datum/reagent/fuel = 1)
+
+/datum/chemical_reaction/welding_fuel
+	name = /datum/reagent/fuel
+	id = /datum/reagent/fuel
+	results = list(/datum/reagent/fuel = 1)
+	required_reagents = list(/datum/reagent/oil = 1, /datum/reagent/toxin/acid = 3)
+	required_temp = 400
+
+/datum/chemical_reaction/stableplasma
+	name = /datum/reagent/stable_plasma
+	id = /datum/reagent/stable_plasma
+	results = list(/datum/reagent/stable_plasma = 5)
+	required_reagents = list(/datum/reagent/toxin/plasma = 3, /datum/reagent/hydrogen = 3)
+	required_catalysts =  list(/datum/reagent/consumable/ethanol = 5)
+	required_temp = 100
+	is_cold_recipe = TRUE
+
+/datum/chemical_reaction/waterformation
+	name = /datum/reagent/water
+	id = /datum/reagent/water
+	results = list(/datum/reagent/water = 1)
+	required_reagents = list(/datum/reagent/oxygen = 1, /datum/reagent/hydrogen = 2)
+	required_temp = 380
+
+/datum/chemical_reaction/ice
+	name = /datum/reagent/consumable/ice
+	id = /datum/reagent/consumable/ice
+	results = list(/datum/reagent/consumable/ice = 2)
```

Comment:
these result numbers not matching the input numbers may have a purpose but it also makes getting the amount of reagent you want with the beaker/dispensing sizes you have more of a pain in the ass

Also it'll be really fun having a disclaimer on every one on the wiki saying "makes x units instead of the input amount"

Reply:
(Addressing multiple in one comment)

in the case of stuff like water, it's because that'd be forming a molecule of it directly
water is 1 oxygen and 2 hydrogen atoms so
it fits

Stable Plasma was indeed ghetto-chem initially, but that was during testing when I was paying with the idea of T-4 stable plasma, but decided against it

Oil being T1 is to save a lot of time, since there's not many ways to get oil other than using its incredibly wonky recipe. Fuel staying T2 was a balance gesture since, it HAD been moved for a reason.

Acid reaction is there because it's now tier locked, so you can still get it without upgrades

Sugar is the same too